package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "react-native-redash",
+  "private": true,
```

Comment:
Won't this prevent me from publishing? Do we have to move redash in `/packages`?